package.json

```diff
@@ -37,7 +37,9 @@
   },
   "homepage": "https://github.com/donmccurdy/aframe-physics-system#readme",
   "dependencies": {
+    "ammo-debug-drawer": "github:infinitelee/ammo-debug-drawer",
     "cannon": "github:donmccurdy/cannon.js#v0.6.2-dev1",
```

Comment:
Can leave this for another time, but I suppose we should pull cannonjs out of the build.